examples/http-streaming/http_streaming.bal

```diff
@@ -1,80 +1,36 @@
 import ballerina/http;
 import ballerina/io;
-import ballerina/log;
 import ballerina/mime;
 
 // Creates an endpoint for the [client](https://docs.central.ballerina.io/ballerina/http/latest/clients/Client).
 http:Client clientEndpoint = check new ("http://localhost:9090");
 
 service /'stream on new http:Listener(9090) {
 
-    resource function get fileupload(http:Caller caller) {
+    resource function get fileupload(http:Caller caller) returns error? {
         http:Request request = new;
 
         //[Sets the file](https://docs.central.ballerina.io/ballerina/http/latest/classes/Request#setFileAsPayload) as the request payload.
         request.setFileAsPayload("./files/BallerinaLang.pdf",
             contentType = mime:APPLICATION_PDF);
 
         //Sends the request to the client with the file content.
-        http:Response|error clientResponse =
-            clientEndpoint->post("/stream/receiver", request);
+        http:Response clientResponse =
+            check clientEndpoint->post("/stream/receiver", request);
```

Comment:
We can return from here right, if we change the return type to `http:Response|error` and remove the `Caller`?